force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -12733,6 +12733,54 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Label to show Elevate Gateways and Payment Methods.</shortDescription>
         <value>Show Elevate Gateways and Payment Methods</value>
     </labels>
+    <labels>
+        <fullName>psEnableGatewayAssignment</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label to show Gateway Assignment is enabled.</shortDescription>
+        <value>Enable Gateway Assignment</value>
+    </labels>
+    <labels>
+        <fullName>psEnableGatewayAssignmentHeader</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Gateway Assignment Header.</shortDescription>
+        <value>Enable Gateway Assignment</value>
+    </labels>
+    <labels>
+        <fullName>psEnableGatewayAssignmentHelp</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Gateway Assignment Help Message.</shortDescription>
+        <value>Enabling gateway assignment will allow you to select the payment gateway and respective payment methods when modifying Gift Entry Templates.</value>
+    </labels>
+    <labels>
+        <fullName>psGatewayManagementHelp</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Gateway Management Help Message.</shortDescription>
+        <value>These settings manage the payment gateways for Elevate transactions.</value>
+    </labels>
+    <labels>
+        <fullName>psGatewayIDHeader</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Gateway ID Header.</shortDescription>
+        <value>Gateway ID</value>
+    </labels>
+    <labels>
+        <fullName>psGatewayIDHelp</fullName>
+        <categories>Payment Gateway Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Gateway ID Help Message.</shortDescription>
+        <value>NPSP uses the gateway that was added to Elevate most recently unless you override it. To change the gateway NPSP usess, click Edit, enter the gateway ID, and save your work. Find the gateway ID in the Elevate Admin Console | Gateway Management.</value>
```

Comment:
Is this correct @npsp-reedestockton? I thought users were no longer going to need to go to the Elevate Admin Console to find the Gateway ID and then copy it and paste it into NPSP settings. If it is not accurate, please use the text approved by the UX Text team: "NPSP uses the gateway that was added to Elevate most recently unless you override it. To set a default Elevate payment gateway, select it here."

Reply:
@ErinWiedemer @screcco-sfdo The text approved by the UX Text team is more accurate. We will swap in the widget to select gateways with no copy/paste after this work item is merged. That work is already done, so it is definitely going to happen. We just need to sequence the work items with this one merging first.

Reply:
Gotcha, thanks!

Reply:
left as is